back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/native_survey_not_submitted_mailer_preview.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module EmailCampaigns
+  class NativeSurveyNotSubmittedMailerPreview < ActionMailer::Preview
+    include EmailCampaigns::MailerPreviewRecipient
+
+    def campaign_mail
+      project = Project.first
+      command = {
+        recipient: recipient_user,
+        event_payload: {
+          survey_url: Frontend::UrlService.new.model_to_url(project, locale: recipient_user.locale),
+          phase_title_multiloc: project.phases.first.title_multiloc || project.title_multiloc,
+          phase_end_at: project.created_at
+        }
+      }
```

Comment:
Do you think we could call `Campaigns::NativeSurveyNotSubmitted#generate_command` here? Currently, it's reimplemented, and in the future, it'll lead to a discrepancy with `generate_command` (I noticed it a few times working on [this PR](https://github.com/CitizenLabDotCo/citizenlab/pull/6525/files#diff-4386f66422ad2840d7a0f8b00f514fb4bdd66511a03843375226a4cd3305948eR11)), which means the preview will behave differently than in production. 

And the same about [the spec](https://github.com/CitizenLabDotCo/citizenlab/pull/6928/files#diff-15272ec3ead22faed3301e4afdae4b98f5bf761e91fc6d23088673c8b6932e48R11).